bin/incredible-squaring-avs/src/commands/avs/mod.rs

```diff
@@ -452,13 +454,13 @@ pub async fn register_operator_with_el_and_avs(
         hex::encode(s).to_string(),
     );
 
-    let operator_details = Operator::new(
-        signer.address(),
-        signer.address(),
-        Address::ZERO,
-        200,
-        Some("url".to_string()),
-    );
+    let operator_details = Operator{
+        address:signer.address(),
+        earnings_receiver_address: signer.address(),
+        delegation_approver_address:Address::ZERO,
+        staker_opt_out_window_blocks:200,
+        metadata_url:Some("url".to_string()),
```

Comment:
I see this PR is ok, the only little comment I have is that this seems not to be formated with `cargo fmt`

Reply:
done